tests/rt.conf

```diff
@@ -110,7 +110,7 @@ RUN | cpld_warmstart_c12_9deg                           | - noaacloud
 RUN | cpld_restart_c12_9deg                             | - noaacloud                          |          | cpld_warmstart_c12_9deg
 
 COMPILE | s2sw_pdlib_debug | intel | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8_ugwpv1 -DPDLIB=ON -DDEBUG=ON | - noaacloud | fv3 |
-RUN | cpld_debug_pdlib_p8                               | - noaacloud                          | baseline |
+RUN | cpld_debug_pdlib_p8                               | - noaacloud orion hercules           | baseline |
```

Comment:
@RatkoVasic-NOAA here is one test that should be attempted to be turned back on with the new build of spack stack for both orion and hercules.

Reply:
Done.